crates/mdb/src/lib.rs

```diff
@@ -132,7 +133,7 @@ pub struct SubscriberConfig {
 impl SubscriberConfig {
     pub fn try_new<F>(topic: &CStr, source: &CStr, on_message: F) -> Result<Self, Error>
     where
-        F: for<'a> FnMut(Message<'a>) + Send + 'static,
+        F: for<'a> FnMut(&Message<'a>) + Send + 'static,
```

Comment:
Should this change?

Reply:
I wasn't sure, I thought for cohesiveness it'd be nice if "callbacks are passed shared references" even though `Message` already is, at least in essence, a reference. It does look a little weird that the lifetime is on `Message` and not `&` though. Ideally it would be a `&'a Message`, but maybe that's something to consider in a different PR, I'll remove it here.